.npmignore

```diff
@@ -22,8 +22,9 @@ lighthouse-logger/
 !lighthouse-core/scripts/manual-chrome-launcher.js
 !lighthouse-core/scripts/download-chrome.sh
 
-# keep smokehouse tests, etc
-!lighthouse-cli/test
```

Comment:
was this a noop previously? it seems like we never actually said to ignore this folder, so why did we negate it... 🤔

Reply:
I think it was just to make it explicit, more important when release was just a checklist, not a script, so missing tests might not be noticed.